source/fuzz/fuzzer_util.cpp

```diff
@@ -520,6 +522,27 @@ SpvStorageClass GetStorageClassFromPointerType(
       pointer_type_inst->GetSingleWordInOperand(0));
 }
 
+std::vector<std::vector<uint32_t>> GetConstantWordsFromUniformsForType(
```

Comment:
As mentioned above, I'm not convinced by the need for this method in here (feel free to convince me).